src/Components/SearchBar.jsx

```diff
@@ -73,7 +80,26 @@ export default function SearchBar({onClickMenuCb, showNavPanel}) {
         value={inputText}
         onChange={onInputChange}
         placeholder={'Search model'}/>
-      <FormButton title='search' icon={<SearchIcon/>}/>
+      <TooltipToggleButton
+        title='clear'
+        size = 'small'
+        placement = 'bottom'
+        onClick={()=>setInputText('')}
+        icon={<ClearIcon/>}/>
+      <FormButton
+        title='search'
+        size = 'small'
+        placement = 'bottom'
+        icon={<SearchIcon/>}/>
+      <TooltipToggleButton
+        title={`Type GitHUB URL to access IFCs hosted on GitHUB.
+                Click on the link icon to learn more.`}
+        size = 'small'
+        placement = 'right'
+        onClick={()=>{
+          window.open('https://github.com/bldrs-ai/Share/wiki/GitHub-model-hosting')
```

Comment:
What did you make of the idea to pop-up a url-entry dialog.. demo link isn't working so not sure if you did.

I'd have assumed it would be here?

Reply:
check the link. 
Dialog is not so straight forward since the link icon is within search input component.